gradio/blocks.py

```diff
@@ -1105,7 +1105,6 @@ def get_config_file(self):
             "show_api": self.show_api,
             "is_colab": utils.colab_check(),
             "stylesheets": self.stylesheets,
-            "root": self.root,
```

Comment:
Did you mean to get rid of this?

Reply:
oh whoops, did not resolve merge conflict correctly